components/Slider/tests/snapshot/MDCSliderSnapshotTests.m

```diff
@@ -141,11 +141,21 @@ - (void)testDynamicColorSupport {
           }
         }];
 
+    UIColor *sliderThumbShadowDynamicColor =
+        [UIColor colorWithDynamicProvider:^(UITraitCollection *traitCollection) {
+          if (traitCollection.userInterfaceStyle == UIUserInterfaceStyleLight) {
+            return UIColor.blackColor;
+          } else {
+            return UIColor.clearColor;
```

Comment:
Should we do this a color that we can see, like blue?

Reply:
Will add one. The reason I didn't add initially was because we don't apply color to shadow in the dark mode by default.

Reply:
Addressed in 17259b9d4826365387da3694fef5d2e2d3fb0542